view_user_birthdate.py

```diff
@@ -0,0 +1,242 @@
+"""Build a dictionary of usernames and their corresponding birthdates
+formatted to match their respective nationalities. Additionally, take
+in user input, match it to any usernames in the dictionary and return
+that user's formatted birthdate.
+"""
+import re
+from get_input import get_input
+
+class User(object):
+    """ Take in two strings 'username' and 'nationality' and a Date
+    'birthdate'. Create an object of type User with those same
+    variables as attributes. Keep track of all instantiated User
+    objects with the dictionary 'bday_book'.
+    """
+    bday_book = {}
```

Comment:
this class variable can't be reused- if i create a User anywhere in my system, the *global bday_book variable* gets updated, no matter where I am. this means parts of my program that have nothing to do with each other are *coupled* needlessly. move this guy out